style.css

```diff
@@ -0,0 +1,41 @@
+.calculator{
+    width: 25%;
+    height: 40%;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-auto-rows: minmax(30px, 80px);
+    grid-gap: 5px;
+    font-size: 1.5em;
+    
+}
+
+.calculator .operator{
+    font-size: 1.25em;
+}
+
+.calculator .zero{
+    grid-column: span 2;
+    border-radius: 5%;
+}
+
+.calculator .number{
+    border-radius: 25px;
+}
+
+.button:hover {
```

Comment:
Consider adding `cursor: pointer;` here as a hint that the user is meant to click the buttons.